docs/INSTALLATION.md

```diff
@@ -0,0 +1,89 @@
+# Installation
+## Overview
+We understand you will probably be excited to power on your AmpliPro and try it out. To avoid damaging the unit, please read through this guide before installing and powering your AmpliPro unit!
+
+WARNING!! VERIFY THAT THE VOLTAGE OF YOUR WALL OUTLET MATCHES THE VOLTAGE LISTED NEAR THE POWER INLET BEFORE PLUGGING IN.
+
+See the power section for more details on how to change input voltage.
```

Comment:
We can not instruct our customers to open up the unit and change the power settings